### PR TITLE
Elor/feature/frontend/waterfall view

### PR DESCRIPTION
**Pull request by Elor170**
#7 and logs for this issue.

### CODE REVIEW

File: packages/hmi-helper/src/methods.ts
Comment (TheDokT0r) [inline packages/hmi-helper/src/methods.ts:1]: I think the file name is a bit too general. Can you make it more specific to what the file does?

File: packages/hmi-helper/src/vars.ts [hunk 1]
```diff
@@ -1 +1,3 @@
 export const sendingIntervalValues = [10, 100, 500, 1000, 5000, 10000] as const;
+export const screenHeight = 1940;
```
Comment (TheDokT0r) [inline packages/hmi-helper/src/vars.ts:2]: screen height of what? the entire app? Why's it in the hmi-helper lib?

File: packages/waterfall-streamer/src/Utilities/dataGenerator.ts [hunk 2]
```diff
@@ -2,24 +2,48 @@ type WaterfallDataType = 'constant' | 'random';
 let dataType: WaterfallDataType = 'constant';
 
 let counter: number = 0;
-// TODO: return the default value
-// const arraysLength: number = 4096;
-const arraysLength: number = 256;
+const arraysLength: number = 3840;
```
Comment (TheDokT0r) [inline packages/waterfall-streamer/src/Utilities/dataGenerator.ts:5]: rather a small detail, but this vars are all consts. I'm nore sure you really need to state their types.

Comment (Elor170) [reply packages/waterfall-streamer/src/Utilities/dataGenerator.ts:5]: So just change this to 
```suggestion
const arraysLength = 3840;
```

File: packages/hmi-helper/src/methods.ts
Comment (TheDokT0r) [reply packages/hmi-helper/src/methods.ts:1]: You defensively can. But in another note, why is this function in hmi-helper? Do you use it in more then one microservice?

Comment (Elor170) [reply packages/hmi-helper/src/methods.ts:1]: The problem I use sendingIntervalValues in this function.

Comment (TheDokT0r) [reply packages/hmi-helper/src/methods.ts:1]: then can you import the variable in the package you're using the function in, and write the function there?

**Pull request merged**